.database_generator/mangos_translation/lightshope.py

```
@@ -0,0 +1,325 @@
+
+
+def download_lightshope() -> None:
+  """Download the LightsHope database ZIP file into the temporary locales directory."""
+  url = "https://github.com/Questie/QuestieDB/releases/download/LightsHope-db/world_full_14_june_2021.zip"
+  output_dir = "locales"
+  os.makedirs(output_dir, exist_ok=True)
+  output_file = os.path.join(output_dir, "world_full_14_june_2021.zip")
+
+  try:
+    print(f"Downloading from {url} to {output_file} ...")
+    response = requests.get(url, stream=True)
```

Ruff (0.14.6)

Probable use of `requests` call without timeout

(S113)